tests/ops_test.py

```diff
@@ -168,35 +179,39 @@ def test_nufft1_vmap(ndim, num_nonnuniform, num_uniform, iflag):
     c = c.astype(cdtype)
     func = partial(nufft1, num_uniform, iflag=iflag)
 
-    # Start by checking the full basic vmap
-    calc = jax.vmap(func)(c, *x)
-    for n in range(num_repeat):
-        np.testing.assert_allclose(calc[n], func(c[n], *(x_[n] for x_ in x)))
-
-    # With different in_axes
-    calc_ax = jax.vmap(func, in_axes=(1,) + (0,) * ndim)(jnp.moveaxis(c, 0, 1), *x)
-    np.testing.assert_allclose(calc_ax, calc)
-
-    # With unmapped source axis
-    calc_unmap = jax.vmap(func, in_axes=(None,) + (0,) * ndim)(c[0], *x)
-    for n in range(num_repeat):
-        np.testing.assert_allclose(calc_unmap[n], func(c[0], *(x_[n] for x_ in x)))
-
-    # With unmapped points axis
-    calc_unmap_pt = jax.vmap(func, in_axes=(0,) + (0,) * (ndim - 1) + (None,))(
-        c, *x[:-1], x[-1][0]
-    )
-    for n in range(num_repeat):
-        np.testing.assert_allclose(
-            calc_unmap_pt[n], func(c[n], *(x_[n] for x_ in x[:-1]), x[-1][0])
+    with jax.experimental.enable_x64():
```

Comment:
Do we need this here? Perhaps we could adjust the tolerance for the allclose calls below to test both? I think the `jax._src.public_test_util.check_close` might do what we want.